scripts/generate_tutorial_readmes.py

```diff
@@ -93,6 +93,12 @@ def main():
         course_readme_text.extend(write_badge_table(student_notebooks))
         course_readme_text.append("\n")
 
+        # Add further reading
+        further_reading_file = f"{day_path}/further_reading.md"
+        if os.path.exists(further_reading_file):
+            course_readme_text.extend([f"[Further Reading](https://github.com/NeuromatchAcademy/course-content/blob/master/{further_reading_file})"])
```

Comment:
This needs to use abstraction over the different course URLs and GitHub branch names. See the global variables at the top of the script.

Reply:
Also stylistically the line is quite long so breaking it up (defining the url, then interpolating that into the full markdown string) would be better.